### ISSUE TITLE
M1 Mac Support

### ISSUE THREAD

**Issue by ghost** [mbruggmann/osx-edid-overrides#7]
Thank you for the excellent script - worked flawlessly on my girlfriend's 2019 Macbook 16".

I've recently purchased one of the new M1 Macbook Airs, and it looks like the display info is stored in a different location. Any idea on how to update the script to make it work?

**Comment by mbruggmann** [comment]
No, sorry, I don't have access to an M1 machine at the moment. But if someone finds out, happy to incorporate it here!

**Comment by aednichols** [comment]
@opentermt please consider filing a report through the Mac Feedback Assistant describing the YPbPr problem and referencing my existing report FB8946046.

Be sure to have the problematic monitor connected at the time of submission so that it shows up in the Sysdiagnose that is collected as part of the report.

**Comment by sam-moshenko** [comment]
Found another way to fix EDID https://forums.macrumors.com/threads/mac-mini-4k-120hz.2267035/page-31?post=31952813#post-31952813
